install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -15,8 +15,8 @@
       {{- include "nodeaffinity" . | indent 6 }}
       containers:
       - name: mixer
-{{- if contains "/" .Values.image }}
-        image: "{{ .Values.image }}"
+{{- if contains "/" $.Values.image }}
```

Comment:
Not sure what this does - but the previous version seemed to work.

Reply:
This should be a bug, but not triggered. 
Anyway, I will create another PR to fix this.